leetcode/string.md

```diff
@@ -9,6 +9,27 @@
 
```

Comment:
Your Pull Request title: LEETCODE-group-anagrams

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']
* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']